bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/utils/AudioStreamUtils.java

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
Whats is the reason for this class? We have `org.apache.commons.io` already in the classpath which provides `FilenameUtils`. The methods are identical to these implemented here.